web-js/add-timestamp-to-wms-request/README.md

```diff
@@ -0,0 +1,31 @@
+#Add Timestamp to WMS Requests
+
```

Comment:
nevermind, the .gif in this PR is from the upload shapefile sample.  including another for the WMS sample, would be cool too, but not mandatory.